workflow/troubleshooting/general.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: "General"
+sidebarTitle: "General"
+---
+
+## Running Steps Inside try/catch Blocks
+
+When running steps (run, sleep, call, or any other step) inside a try/catch block, you'll encounter the following error:
+
+```
+WorkflowAbort: This is an Upstash Workflow error thrown after
+a step executes. It is expected to be raised. Make sure that
+you await for each step. Also, if you are using try/catch
+blocks, you should not wrap context.run/sleep/sleepUntil/call
+methods with try/catch. Aborting workflow after executing
+step 'some-step'.
+```
+
+The `WorkflowAbort` error is thrown by the Workflow SDK when a step executes successfully. When you run a step inside a try/catch block, you have two primary options:
```

Comment:
Doesn't it make sense to make the user use the try/catch inside the `context.run` block? Maybe we can advise them that way

Reply:
updated